tests/system/small/test_remote_function.py

```diff
@@ -839,6 +839,89 @@ def test_read_gbq_function_enforces_explicit_types(
         )
 
 
+@pytest.mark.flaky(retries=2, delay=120)
+def test_read_gbq_function_row_processor_error(
+    session,
+    bigquery_client,
+    bigqueryconnection_client,
+    cloudfunctions_client,
+    resourcemanager_client,
+    dataset_id_permanent,
+    bq_cf_connection,
+):
+    def func_rp0(num0: int, num1: int) -> int:
+        return num0 + num1
+
+    func_rp0 = rf.remote_function(
+        bigquery_client=bigquery_client,
+        bigquery_connection_client=bigqueryconnection_client,
+        dataset=dataset_id_permanent,
+        cloud_functions_client=cloudfunctions_client,
+        resource_manager_client=resourcemanager_client,
+        bigquery_connection=bq_cf_connection,
+        reuse=True,
+        name=get_rf_name(func_rp0),
+    )(func_rp0)
+
+    with pytest.raises(ValueError) as context:
+        # The remote function has two args, which cannot be row processed. Throw
+        # a ValueError for it.
+        rf.read_gbq_function(
+            function_name=func_rp0.bigframes_remote_function,  # type: ignore
+            is_row_processor=True,
+            session=session,
+        )
+    assert (
+        str(context.value)
+        == "The row processor cannot accept a multi-parameter function."
+    )
+
+
+@pytest.mark.flaky(retries=2, delay=120)
+def test_read_gbq_function_row_processor(
```

Comment:
no need to add a separate test. You could add additional logic in the existing test `test_df_apply_axis_1` to read back the deployed remote function and make sure that it works as expected. See for an example https://github.com/googleapis/python-bigquery-dataframes/blob/be74b99977cfbd513def5b7e439de6b7706c0712/tests/system/large/test_remote_function.py#L1607

Reply:
Removed the test and added related functionality in test_df_apply_axis_1.